ark/utils/misc_utils.py

```diff
@@ -135,6 +135,8 @@ def verify_in_list(**kwargs):
         raise ValueError("You must provide 2 arguments to verify_in_list")
 
     test_list, good_values = kwargs.values()
+    if len(test_list) == 0 or len(good_values) == 0:
+        raise ValueError("List arguments cannot be empty")
```

Comment:
Were we running into issues with empty lists, or is this simply an extra error check?

Reply:
See Noah's comment above